resources/linux/bin/nerdctl

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec /mnt/wsl/rancher-desktop/bin/nerdctl "$@"
```

Comment:
This should go into a different directory, like `resources/wsl/bin`, to reduce the friction when we do a linux port.

Reply:
I think we need a bigger change when we do a Linux port, since things that are currently in the `linux` directory are all WSL tools (such as `wsl-helper`).  As for nerdctl, we [already put it there](https://github.com/rancher-sandbox/rancher-desktop/blob/4073ff06591bb26144841bdfc80d33f6b8808536/scripts/lib/build-utils.mjs#L250) (that's on the main branch).

We should just have a PR to move all of it under `resources/win32/wsl/bin` or something, but distinct from this one.

Reply:
Scrap the different directory. But on my windows box, this file came down with CRLF line endings, which doesn't work for shell scripts invoked in a WSL subsystem.  Fix is to add a .gitattributes file which ensures this file's EOLs are LF only.

Reply:
Yep, the `.gitattributes` was added in dfd4331f7255becc53eaa0429611cdacd4a374f0 — I just had you try it first to ensure that the fix works (since my configuration is weird and wouldn't reproduce the issue properly).